src/main/java/dev/openfeature/sdk/MutableStructure.java

```diff
@@ -34,55 +37,103 @@ public Set<String> keySet() {
         return this.attributes.keySet();
     }
 
-    // getters
+
     @Override
     public Value getValue(String key) {
         return this.attributes.get(key);
     }
 
-    // adders
+    /**
+     * Adds the specified value at key.  
```

Comment:
These docs feel kinda useless?

Reply:
> If you're somehow blocked on jdk versions.. I'd like to separate that from the documentation changes. I think the documentation changes are of low value (no offense intended) and I don't really want to accept them. No one needs to know that collection.add(thing) adds the thing to the collection. It's apparent on it's face.

I see your point. While these ones in particular aren't super valuable, even in this PR there are some that are (I think some of the exceptions are nice, for instance).

I guess the questions is, would you rather see us add this rule, and ignore it where we think it's silly, or not add the rule, and be careful to add doc where users might be confused.

Reply:
My preference would be to enable the rule and explicitly say "skip this one" on the things that are very obvious to keep the tool happy.

Reply:
ACK!

Reply:
Hey @justinabrahms I've done this.

I removed the lowest value javadocs (like the ones for the adders) and added the capability to suppress checkstyle warnings with annotations for those instances.